opensearch-operator/Dockerfile.oracle

```diff
@@ -27,10 +27,10 @@ RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -a -o manager m
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
```

Comment:
I guess they were using distroless in the upstream dockerfile? I think we should not have a reference to the GoogleContainerTools in our Dockerfile.

Reply:
I was unsure whether to remove this or not. I have gone ahead and removed it.